### PR TITLE
Add pytorch cifar10 ddp example with learning rate scaling

### PR DESCRIPTION
**Pull request by DN6**
This PR:

Adds the Pytorch DDP Example with the CIFAR10 dataset. The example was used in this report
https://www.comet.ml/team-comet-ml/parallelism/reports/advanced-ml-parallelism

Updates the README instructions with a reference to the new example.

### CODE REVIEW

**Comment by dsblank** [comment]
Nice! Would it make sense to add a link to the report in the README?

**Comment by DN6** [comment]
@dsblank  Great idea. Can do!

**Review by dsblank — APPROVED** [review]

**Comment by dsblank** [comment]
Thanks!
